README.md

```diff
@@ -1,3 +1,148 @@
 # Hapi Bookshelf Total Count
 
-Hapi plugin used with Bookshelf models to calculate the total number of records that match a query and appends it to the response.
+A [Hapi](http://hapijs.com/) plugin used with [Bookshelf](http://bookshelfjs.org/) models to calculate the total number of records that match a query and appends it to the response. It can be used to calculate an [absolute total count](#absolute-total-count) or a [filtered total count](#filtered-total-count).
+
+## Absolute Total Count
+
+Appends the `total_count` of all instances of the model when the query contains `include[]=total_count`, regardless of query filter.
+
+### Register the Plugin
+
+```js
+const Hapi = require('hapi');
+
+const server = new Hapi.Server();
+
+server.register([
+  require('hapi-bookshelf-total-count')
+], (err) => {
+
+});
+```
+
+### Configure the endpoint
+
+```js
+const Bookshelf = require('bluebird')(require('knex')(config));
+
+const Book = Bookshelf.Model.extend({ tableName: 'books' });
+
+server.route({
+  method: 'GET',
+  path: '/books',
+  config: {
+    plugins: {
+      totalCount: { model: Book }
+    },
+    handler: (request, reply) => {
+      return new Book().fetchAll()
+      .then((books) => {
+        reply({ data: books });
+      });
+    }
+  }
+});
+
+```
+
+### Request
+
+```bash
+$ curl -g GET "https://YOUR_DOMAIN/books?include[]=total_count"
```

Comment:
Is the `-g` necessary? I thought that if the `[]` is in quotes that it should be fine.